mod/peer/gossip/state/state.go

```diff
@@ -77,7 +77,7 @@ type GossipServiceMediator interface {
 }
 
 //AddBlockHandler handles state update in gossip
-func AddBlockHandler(publisher api.BlockPublisher) {
+func AddBlockHandler(cid string, publisher api.BlockPublisher) {
```

Comment:
What is cid, chaincode ID? Why do we need this?

Reply:
This has to be added when referencing latest fabric-mod. This was added to fabric-mod as part of https://github.com/trustbloc/fabric-mod/pull/78.

Reply:
ok